webapp-django/crashstats/crashstats/jinja2/crashstats/report_index.html

```diff
@@ -825,21 +825,18 @@ <h2>Related Bugs</h2>
           <div id="rawdump" class="ui-tabs-hide">
             <div class="code">{{ raw_stackwalker_output }}</div>
 
-            <h3>Download the Raw Dump</h3>
+            <h3>Download Raw Crash Data and Minidumps</h3>
             {% if request.user.has_perm('crashstats.view_rawdump') %}
+              <p>
+                {{ sensitive_warning(your_crash) }}
```

Comment:
This adds the warning for the entire section. I also tweaked the section to either show the whole thing or not depending on whether the user has the appropriate permissions.